### PR TITLE
Add tool configs + finalize implementation

### PR DESCRIPTION
**Pull request by apfelbox**

### CODE REVIEW

File: _init/next-js/.stylelintrc.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+extends:
+    - './node_modules/@21torr/janus/stylelint/base.yml'
+
+# THIS IS A TEST
```
Comment (keichinger) [inline _init/next-js/.stylelintrc.yml]: Is it though?

Comment (apfelbox) [reply _init/next-js/.stylelintrc.yml]: It is and you passed it.

File: dprint/app.json [hunk 2]
```diff
@@ -1,6 +1,14 @@
 {
 	"lineWidth": 120,
 	"useTabs": true,
+	"prettier": {
+		"associations": [
+			"**/*.{css,scss}"
+		],
```
Comment (keichinger) [inline dprint/app.json:7]: Do we need to ignore `node_modules`?

File: _init/next-js/package.json [hunk 3]
```diff
@@ -1,9 +1,17 @@
...
 	"devDependencies": {
-		"dprint": "^0.45.0"
+		"@typescript-eslint/eslint-plugin": "^6.18.1",
+		"dprint": "^0.45.0",
+		"eslint": "^8.56.0",
+		"eslint-plugin-jsdoc": "^48.0.2",
+		"eslint-plugin-react": "^7.33.2",
+		"eslint-plugin-react-hooks": "^4.6.0",
+		"eslint-plugin-storybook": "^0.6.15",
+		"stylelint": "^16.1.0",
```
Comment (keichinger) [inline _init/next-js/package.json]:
```suggestion
		"stylelint": "^16.4.0",
```

File: dprint/app.json (hunk 2 above)
Comment (apfelbox) [reply dprint/app.json:7]: It's already ignored, see `excludes` below

File: _init/next-js/package.json (hunk 3 above)
Comment (apfelbox) [reply _init/next-js/package.json]: Now you see since when the changes were in my local repository?

**Review by keichinger — APPROVED** [review]

File: _init/next-js/package.json (hunk 3 above)
Comment (keichinger) [reply _init/next-js/package.json]: At least for 4 months 😂

File: dprint/app.json
Comment (21christiansc) [inline dprint/app.json:1]: Should we add the storybook static folder (from building storybook) to the exclude list?

Comment (apfelbox) [reply dprint/app.json:1]: Good idea, I just added it

**Review by keichinger — APPROVED** [review]

**Pull request merged**